pom.xml

```diff
@@ -133,6 +133,15 @@
             <artifactId>jcabi-manifests</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.jcabi</groupId>
+            <artifactId>jcabi-s3</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk</artifactId>
+            <version>1.7.13</version>
```

Comment:
Is there any particular reason why we aren't using a later version here? I believe the AWS Java SDK is now up to 1.9.x.